tests/php/test-amp-img-sanitizer.php

```diff
@@ -698,4 +701,86 @@ public function get_new_attachment_id() {
+</div>
+<div>
+	<picture><source srcset="https://interactive-examples.mdn.mozilla.net/media/cc0-images/surfer-240-200.jpg" media="(min-width: 800px)"><img src="https://interactive-examples.mdn.mozilla.net/media/cc0-images/painted-hand-298-332.jpg??image=3" alt=""></picture>
+</div>';
+
+		return [
+			'allow_picture_false'                     => [
+				'input'    => $content,
+				'args'     => [
+					'allow_picture' => false,
+				],
+				'expected' => '<div>
+	<img src="https://interactive-examples.mdn.mozilla.net/media/cc0-images/painted-hand-298-332.jpg?image=1" alt="">
+	<img src="https://interactive-examples.mdn.mozilla.net/media/cc0-images/surfer-240-200.jpg?image=2" alt="">
+</div>
+<div>
+	<img src="https://interactive-examples.mdn.mozilla.net/media/cc0-images/painted-hand-298-332.jpg??image=3" alt="">
+</div>',
+			],
+			'allow_picture_true'                      => [
+				'input'    => $content,
+				'args'     => [
+					'allow_picture' => true,
+				],
+				'expected' => '<div>
+	<picture data-px-verified-tag><div class="screen-reader-text"></div><source srcset="https://interactive-examples.mdn.mozilla.net/media/cc0-images/surfer-240-200.jpg" media="(min-width: 800px)" data-px-verified-tag><img src="https://interactive-examples.mdn.mozilla.net/media/cc0-images/painted-hand-298-332.jpg?image=1" alt="" data-px-verified-tag></picture>
+	<picture data-px-verified-tag><source srcset="https://interactive-examples.mdn.mozilla.net/media/cc0-images/surfer-240-200.jpg" media="(min-width: 800px)" data-px-verified-tag><img src="https://interactive-examples.mdn.mozilla.net/media/cc0-images/surfer-240-200.jpg?image=2" alt="" data-px-verified-tag></picture>
+</div>
+<div>
+	<picture data-px-verified-tag><source srcset="https://interactive-examples.mdn.mozilla.net/media/cc0-images/surfer-240-200.jpg" media="(min-width: 800px)" data-px-verified-tag><img src="https://interactive-examples.mdn.mozilla.net/media/cc0-images/painted-hand-298-332.jpg??image=3" alt="" data-px-verified-tag></picture>
+</div>',
+			],
+			'without_picture_element'                 => [
+				'input'    => '<h1>Page heading</h1><ul><li>Item 1</li><li>Item 2</li></ul>',
+				'args'     => [
+					'allow_picture' => true,
+				],
+				'expected' => '<h1>Page heading</h1><ul><li>Item 1</li><li>Item 2</li></ul>',
+			],
+			'picture_without_img_allow_picture_false' => [
+				'input'    => '<picture><div class="screen-reader-text"></div><source srcset="https://interactive-examples.mdn.mozilla.net/media/cc0-images/surfer-240-200.jpg" media="(min-width: 800px)"></picture>',
+				'args'     => [
+					'allow_picture' => false,
+				],
+				'expected' => '<picture><div class="screen-reader-text"></div><source srcset="https://interactive-examples.mdn.mozilla.net/media/cc0-images/surfer-240-200.jpg" media="(min-width: 800px)"></picture>',
```

Comment:
Since `allow_picture` is `false`, why isn't the `picture` converted? Oh, it's because there is no `img`?